src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -3812,7 +3812,7 @@ public void CscUtf8Output_WithRedirecting_On()
 
             var output = RunAndGetOutput("cmd", "/C \"" + s_CSharpCompilerExecutable + "\" /utf8output /nologo /preferreduilang:en /t:library " + srcFile + " > " + tempOut.Path, expectedRetCode: 1);
             Assert.Equal("", output.Trim());
-            Assert.Equal("SRC.CS(1,1): error CS1056: Unexpected character '♚'", tempOut.ReadAllText().Trim().Replace(srcFile, "SRC.CS"));
+            Assert.Equal("SRC.CS(1,1): error CS1056: Unexpected character '\u265A'", tempOut.ReadAllText().Trim().Replace(srcFile, "SRC.CS"));
```

Comment:
@CyrusNajmabadi the default behavior in codeformatter is to escape unicode characters. I neglected to turn the behavior off when I did this run.